test/core/rebalancingSetToken.spec.ts

```diff
@@ -302,6 +352,63 @@ contract('RebalancingSetToken', accounts => {
       assertTokenBalance(rebalancingSetToken, expectedNewBalance, subjectIssuer);
     });
 
+    describe('when the caller is not Core', async () => {
+      beforeEach(async () => {
+        subjectCaller = otherAccount;
+      });
+
+      it('should revert', async () => {
+        await expectRevertError(subject());
+      });
+    });
+  });
+
+  describe('#mint: Called from CoreMock', async () => {
+    let rebalancingSetToken: RebalancingSetTokenContract;
+    let subjectIssuer: Address;
+    let subjectQuantity: BigNumber;
+    let subjectCaller: Address;
+
+    let newRebalancingSetToken: SetTokenContract;
+
+    beforeEach(async () => {
+      const setTokens = await rebalancingTokenWrapper.createSetTokens(coreMock, factory.address, transferProxy.address);
```

Comment:
You could combine this and the next two lines in one line
```
const [ currentSetToken, newRebalancingSetToken ] = await rebalancingTokenWrapper.createSetTokens...
```

Reply:
Different scoping on them though